tests/integration/test_recipe.py

```diff
@@ -823,25 +818,7 @@ def test_alias_generation(tmp_path, patched_datafinder, config_user):
     diag = recipe.diagnostics['diagnostic_name']
     var = diag['preprocessor_output']['ta']
     for dataset in var:
-        if dataset['project'] == 'CMIP5':
-            if dataset['dataset'] == 'GFDL-CM3':
-                assert dataset['alias'] == 'CMIP5_GFDL-CM3'
-            else:
-                if dataset['ensemble'] == 'r1i1p1':
-                    assert dataset['alias'] == 'CMIP5_EC-EARTH_r1i1p1'
-                elif dataset['ensemble'] == 'r2i1p1':
-                    assert dataset['alias'] == 'CMIP5_EC-EARTH_r2i1p1'
-                else:
-                    assert dataset['alias'] == 'custom_alias'
-        elif dataset['project'] == 'CMIP6':
-            if dataset['dataset'] == 'GFDL-CM3':
-                assert dataset['alias'] == 'CMIP6_GFDL-CM3'
-            elif dataset['dataset'] == 'EC-EARTH':
-                assert dataset['alias'] == 'CMIP6_EC-EARTH'
-            else:
-                assert dataset['alias'] == 'CMIP6_HADGEM'
+        if dataset['exp'] == 'historical':
```

Comment:
It looks like you are loosing many test cases, wouldn't it make sense to keep those too?

Reply:
Thanks. Now it is fixed